src/app/passes/pass-list/pass-table-row/pass-table-row.component.ts

```diff
@@ -27,6 +29,56 @@ export class PassTableRowComponent extends TableRowComponent implements OnInit {
   }
 
   navigate(route) {
-    this.router.navigate(['../', 'pass', this.rowData._id, route], { relativeTo: this.route });
+    switch (route) {
+      case 'details':
+        let message = `<strong>First Name:</strong></br>` + this.rowData.firstName;
+        message += `</br></br><strong>Last Name:</strong></br>` + this.rowData.lastName;
+        message += `</br></br><strong>Email:</strong></br>` + this.rowData.email;
+        if (this.rowData.phoneNumber) {
+          message += `</br></br><strong>Phone Number:</strong></br>` + this.rowData.phoneNumber;
+        }
+        if (this.rowData.facilityType === 'Parking') {
+          message += `</br></br><strong>License Plate:</strong></br>` + this.rowData.license;
+        }
+        message += `<hr><strong>Registration Number:</strong></br>` + this.rowData.registrationNumber;
+        message += `</br></br><strong>Facility Name:</strong></br>` + this.rowData.facilityName;
+        message += `</br></br><strong>Booking Time:</strong></br>` + this.rowData.type;
+        message += `</br></br><strong>Date:</strong></br>` + this.rowData.date;
+        message += `</br></br><strong>Pass Status:</strong></br>` + this.rowData.passStatus;
+
+        this.dialogService
+          .addDialog(
+            ConfirmComponent,
+            {
+              title: 'Details for ' + this.rowData.registrationNumber,
+              message,
+              okOnly: true
+            },
+            { backdropColor: 'rgba(0, 0, 0, 0.5)' }
+          );
+        break;
+      case 'cancel':
+        let cancelMessage = `<p>You are about to cancel pass <strong>${this.rowData.registrationNumber}</strong>.</div>`;
+        cancelMessage += `<p>Are you sure you want to continue?</p>`;
+        this.dialogService
+          .addDialog(
```

Comment:
is it possible to use the bootstrap class `.modal-dialog-scrollable` to allow mobile users to scroll extra tall modals?

Reply:
a61f61f